scout/static/scout/js/filter.js

```diff
@@ -197,3 +197,9 @@ var Filter = {
         }
     }
 };
+
+/* node.js exports */
+if (typeof exports == "undefined") {
```

Comment:
Without some tests using this code, this is adding an untested change.  These exports should be added as needed, rather than in anticipation of a need.